framework/src/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -1680,6 +1680,12 @@ public final String getOption(String name) {
     // TODO I would like to rename getLintOption to hasLintOption
     @Override
     public final boolean hasOption(String name) {
+
+        // Always ignore errors from raw type arguments.
+        // See https://github.com/typetools/checker-framework/issues/635
+        if (name.equals("ignoreRawTypeArguments")) {
```

Comment:
4 checkers call `hasOption("ignoreRawTypeArguments')`.  Also, AnnotatedTypeFactory and QualifiedTypeFactoryAdapter call `hasOption("ignoreRawTypeArguments')`.  (There are no places that call `getOption("ignoreRawTypeArguments", false)`.)  By adding this code here those call sites don't have to be changed.